### ISSUE TITLE
Feature: Add Herobrine

### ISSUE THREAD

**Issue by nullTheCoder** [LopyLuna/Create-Calamity#7]

**Comment by Sascha-T** [comment]
how is this suggestion `invalid`?
herobrine may not be directly related to create but I believe that if we take some liberties we can change that

**Comment by nullTheCoder** [comment]
Not invalid then! We can make Herobrine wear goggles and build random create machines around the world to make them even more valid!

**Comment by Sascha-T** [comment]
we could also make him a ~~threat to society~~ minor annoyance by having him just kill steam engines on chance or something (or temporarely change speeds on rotation speed controllers, temporarely flip directionality of pumps, etc)

**Comment by nullTheCoder** [comment]
Yeah, it will force players to build back up systems which could be kind of fun.
